pkg/node/address.go

```diff
@@ -226,13 +226,18 @@ func GetInternalIPv6(logger *slog.Logger) net.IP {
 	return clone(n.GetNodeInternalIPv6())
 }
 
+// GetNodeIP returns node IP, prioritizing IPv4 if enabled in config
```

Comment:
Eh, let's not add more to `pkg/node`. The package global methods are all sort of deprecated, but obviously still in massive use. But let's not add more, I think you can get away with just using `GetCiliumEndpointNodeIP` (this is semantically the right thing anyway; the KVstore entries are CEP equivalents) and then parsing the IP.

Reply:
`net.IP` to `netip.Addr` is also a conversion 🤷
I know it's a bit inefficient but we can work this out once we're getting rid of the `pkg/node` globals.